workflow-service/src/test/java/com/redhat/parodos/workflow/execution/controller/WorkFlowControllerTest.java

```diff
@@ -231,6 +234,22 @@ public void TestGetStatusWithValidData() throws Exception {
 				.andExpect(MockMvcResultMatchers.status().isOk());
 	}
 
+	@Test
+	void TestGetStatusByProjectIdWithValidData() throws Exception {
```

Comment:
I miss the following test:


- where checks that the project is nill? Does it return all projects?
- What happens is getWorkFlowStatusByProjectId return an empty list?

Reply:
1. yes  it should return all projects which the user is allowed to check
2. if it returns empty list then UI won't show any executions in the workflow table

Reply:
I will add tests for them